### PR TITLE
Adds form to allow user to search for any stock in the api, returns last 60 days of that stock

### PR DESCRIPTION
**Pull request by rbaxter11**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -7,17 +7,9 @@ function App() {
...
-        <p>🚀🚀🚀🚀🚀 stonks 🚀🚀🚀🚀🚀</p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+        <p>Type in a stock symbol below to get the latest close price</p>
```
Comment (adamscottbaxter) [inline src/App.js]: Can you move this into the `Price` component.

File: src/index.js [hunk 2]
```diff
@@ -6,68 +6,105 @@ import reportWebVitals from './reportWebVitals';
...
-//     super(props);
-//     this.state = {
-//       isLoaded: false,
-//       stocks: [],
+class Price extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isLoaded: false,
+      stocks: [],
```
Comment (adamscottbaxter) [inline src/index.js]: Can you rename this to something more descriptive? Like `priceHistory`

File: src/index.js [hunk 3]
```diff
@@ -6,68 +6,105 @@ import reportWebVitals from './reportWebVitals';
...
-//   constructor(props) {
-//     super(props);
-//     this.state = {
-//       isLoaded: false,
-//       stocks: [],
+class Price extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isLoaded: false,
```
Comment (adamscottbaxter) [inline src/index.js:13]: Is `isLoaded` being used?

**Review by adamscottbaxter — CHANGES_REQUESTED** [review]
Looks great. Glad you got back to the other component design. Left a few little comments.

File: src/index.js (hunk 3 above)
Comment (rbaxter11) [reply src/index.js:13]: It isn't being used at the moment, but I'd like to leave it in as a useful tool for later (might be helpful for components down the line as I learn more about React). For now I'll comment it out.

**Pull request merged**